src/test/java/org/mule/tools/apikit/MainAppScaffolderTest.java

```diff
@@ -537,6 +542,46 @@ private void simpleGenerateForCE(final String apiPath) throws Exception {
     assertEquals(5, countOccurences(s, "<logger level=\"INFO\" message="));
   }
 
+  private void simpleGenerateHideConsole(final String apiPath) throws Exception {
```

Comment:
I see that the same api is being parsed for several tests. Try to move common assertions from all the tests to some method.